packages/react/src/components/SheetOverlay/ContentEditable.tsx

```diff
@@ -13,70 +13,75 @@ type ContentEditableProps = Omit<
   allowEdit?: boolean;
 };
 
-class ContentEditable extends React.Component<ContentEditableProps> {
-  lastHtml?: string = undefined;
+const ContentEditable: React.FC<ContentEditableProps> = ({ ...props }) => {
+  const [lastHtml, setLastHTML] = useState("");
+  const root = useRef<HTMLDivElement | null>(null);
+  const { autoFocus, initialContent, onChange } = props;
+  const isFirstRender = useRef(true);
 
-  root: HTMLDivElement | null = null;
-
-  componentDidMount() {
-    const { autoFocus, initialContent } = this.props;
-    if (autoFocus) {
-      this.root?.focus();
+  useEffect(() => {
+    if (autoFocus && root instanceof HTMLDivElement) {
+      root.focus();
     }
-    if (initialContent && this.root) {
-      this.root.innerHTML = initialContent;
+    if (initialContent && root instanceof HTMLDivElement) {
+      root.innerHTML = initialContent;
     }
-  }
+  }, [autoFocus]);
 
-  UNSAFE_componentWillUpdate() {
-    const { initialContent } = this.props;
-    if (initialContent && this.root) {
-      this.root.innerHTML = initialContent;
+  // UNSAFE_componentWillUpdate
+  useEffect(() => {
+    if (isFirstRender.current) {
+      isFirstRender.current = false;
+      if (initialContent && root instanceof HTMLDivElement) {
+        root.innerHTML = initialContent;
+      }
     }
-  }
+  }, [initialContent]);
+
+  const fnEmitChange = useCallback(() => {
+    let html;
 
-  emitChange() {
-    const { onChange } = this.props;
-    const html = this.root?.innerHTML;
-    if (onChange && html !== this.lastHtml) {
+    if (root instanceof HTMLDivElement) {
+      html = root.innerHTML;
+    }
```

Comment:
i forgot this
```
 const fnEmitChange = useCallback(() => {
    const html = root.current?.innerHTML;
    if (onChange && html !== lastHtml) {
      onChange(html || "");
    }
...
```

Reply:
thank you again. I fixed that and another things. + updated packages.

BTW: with node 18 exists some troubles with storybook and postcss

Reply:
I'm still using node 16.13.0 ;)